tox.ini

```diff
@@ -0,0 +1,73 @@
+[tox]
+envlist = bandit,black,isort,flake8,mypy,python3.9,python3.10
+
+[gh-actions]
+python =
+    3.9: python3.9
+    3.10: python3.10
+
+[testenv]
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
+whitelist_externals =
+    make
+    mkdir
+    rm
+passenv = TOX_ENV_DIR
+usedevelop = true
+
+commands =
+    pytest \
+        --cov-config .coveragerc --cov=pyarn --cov-report term \
+        --cov-report xml --cov-report html
```

Comment:
Are xml and html useful?

Reply:
I don't think so, I'll remove both of them.